dev-haskell/ghc-platform/ghc-platform-9.12.3.ebuild

```
@@ -0,0 +1,35 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.8.5.1.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour"
+inherit haskell-cabal
+
+DESCRIPTION="Platform information used by GHC and friends"
+HOMEPAGE="https://gitlab.haskell.org/ghc/ghc/-/tree/master/libraries/ghc-platform#readme"
+SRC_URI="https://downloads.haskell.org/ghc/${PV/_/-}/ghc-${PV}-src.tar.xz"
+
+S="${WORKDIR}/ghc-${PV}/libraries/${CABAL_PN}"
+CABAL_FILE="${S}/${CABAL_PN}.cabal"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+
+RDEPEND=">=dev-lang/ghc-9.0.2:=
+"
+DEPEND="${RDEPEND}
+	>=dev-haskell/cabal-3.4.1.0
+"
+
+CABAL_CORE_LIB_GHC_PV="9.12.2 9.12.3"
```

```suggestion
CABAL_CORE_LIB_GHC_PV="9.12.3"
```

---

I am assuming adding `9.12.2` was a mistake, since there is this:

https://github.com/gentoo-haskell/gentoo-haskell/blob/c638c2b8b7369b06096b26697ba7791be73fc733/dev-haskell/ghc-platform/ghc-platform-9.12.2.ebuild#L28